Storage/tests/System/SignedUrlTest.php

```diff
@@ -54,58 +63,129 @@ public function signedUrls()
      */
     public function testSignedUrl($objectName, array $urlOpts = [])
     {
-        if ($objectName === self::RANDOM_NAME) {
-            $objectName = null;
-        }
         $obj = $this->createFile($objectName);
         $ts = new Timestamp(new \DateTime('tomorrow'));
         return $obj->signedUrl($ts, $urlOpts);
 
-        $this->assertEquals(self::CONTENT, $this->getFile($url));
+        $this->assertEquals(self::CONTENT, $this->getFile($url, $urlOpts));
```

Comment:
Does this assertion get hit with the return statement above?

Reply:
updated, and re-ran the tests to confirm no breakage.